src/test/ui/lifetime-errors/ex3-both-anon-regions-3.stderr

```diff
@@ -1,10 +0,0 @@
-error[E0623]: lifetime mismatch
```

Comment:
For other reviewers: `ex3-both-anon-regions-3.rs` was moved to `ex3-both-anon-regions-one-is-struct-2.rs`